### ISSUE TITLE
sortByPath

### ISSUE THREAD

**Issue by char0n** [char0n/ramda-adjunct#1240]
**Is your feature request related to a problem? Please describe.**

Sorts the list according to the supplied path

**Describe the solution you'd like**

```js
const sortByFirstItem = R.sortByPath([0]);
const pairs = [[-1, 1], [-2, 2], [-3, 3]];
sortByFirstItem(pairs); //=> [[-3, 3], [-2, 2], [-1, 1]]

const sortByName = R.sortByPath(['name']);
const alice = {
  name: 'ALICE',
  age: 101
};
const bob = {
  name: 'Bob',
  age: -10
};
const clara = {
  name: 'clara',
  age: 314.159
};
const people = [clara, bob, alice];
sortByName(people); //=> [alice, bob, clara]
```

**Describe alternatives you've considered**

---

**Additional context**

Original issue: https://github.com/char0n/ramda-adjunct/issues/945

**Comment by Mxchaeltrxn** [comment]
Hey I can try do this

**Comment by Mxchaeltrxn** [comment]
@char0n This seems very similar to sortByProps except that this also works for lists of lists. What exactly is meant by 'path'?

**Comment by char0n** [comment]
@Mxchaeltrxn I've assigned it to you. But first let's focus on https://github.com/char0n/ramda-adjunct/pull/1323. There is still lot of work to be done there. After we're finished there, you'll already know how submit another PR. Thanks.

**Comment by char0n** [comment]
@Mxchaeltrxn there is nothing stopping you to start working on this. Go ahead and good luck!

**Comment by Mxchaeltrxn** [comment]
@char0n I'm a little confused as to why this function partly does the same thing as sortByProps. It makes more sense to me to make it work for lists of lists only because the other functionality was just written

**Comment by char0n** [comment]
@Mxchaeltrxn good point. I've modified the issue description to reflect your comment.
